stable/dex-k8s-authenticator/values.yaml

```diff
@@ -0,0 +1,123 @@
+# Default values for dex-k8s-authenticator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: mintel/dex-k8s-authenticator
+  tag: latest
+  pullPolicy: Always
+
+nameOverride: ""
+fullnameOverride: ""
+#healthCheckPath: ""
```

Comment:
why do you keep it commented out if you use it in charts above?

Reply:
Typo after testing, going to fix that with next commit.